site/docs/v1/tech/apis/two-factor.adoc

```diff
@@ -147,6 +147,18 @@ include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[
 
 include::docs/v1/tech/apis/_two-factor-disable-request-parameters.adoc[]
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Disable Multi-Factor Authentication With a UserId
+[.endpoint]
+.URI
+--
+[method]#DELETE# [uri]#/api/user/two-factor/`{userId}`#
```

Comment:
I audited all of the APIs and this was the only one that I found that was undocumented. @mooreds my understanding of this ticket was to ensure all APIs that take a Request Body are documented. I can't imagine it would provide much in the way of convenience for endpoint users to add this to a bunch of DELETE endpoints, and I couldn't find any evidence to show that URL parameters are more secure than a request body. I would say document the ones we are already exposing. I don't think there would be any concerns about size limits in a request body that takes these two parameters unless it was some kind of bulk operation, in which case it probably already takes a Request Body. Perhaps I am missing something in terms of security and maybe keeping the APIs consistent?

Reply:
okay, if this is a one-off, fine documenting it for backwards compatibility. Might be worth raising the bigger q with the engineering team about any other delete endpoints going forward.

Reply:
I think we can resolve this convo.

Reply:
We do have other APIs that take a request body or query parameters. The two reasons for this are: (you nailed them)

- Overcome request size limits. For example, bulk user delete - appending thousands of `userIds=` to the query string will blow out the request URL. 
- Can be more confidential 

In general, if an API supports using the request body or query parameters, the client lib version will use the body. This is because it is simpler to write, and requires less code maintenance. 

I would generally speaking say we prefer the request body, but I don't think we should deprecate anything since a using a request body on a `DELETE` request may not always be possible for a client. 

Using the request body is a nice to have if you can do it. The w3 spec suggests that `GET` and `DELETE` methods don't take a request body, but in practice this is done frequently in REST because of the limitations mentioned above.